src/django/oar/settings.py

```diff
@@ -177,3 +179,8 @@
 
 # Application settings
 MAX_UPLOADED_FILE_SIZE_IN_BYTES = 5242880
+
+GOOGLE_GEOCODING_API_KEY = os.getenv('GOOGLE_GEOCODING_API_KEY')
+if GOOGLE_GEOCODING_API_KEY is None:
+    raise ImproperlyConfigured(
```

Comment:
Does this effectively guarantee that the application cannot run without the key being set?

If so, I'll dispense with internal unset-key error handling for the work on #77

Reply:
Nevermind, I confirmed that that is indeed the case:

```sh
django_1    |     self._wrapped = Settings(settings_module)
django_1    |   File "/usr/local/lib/python3.7/site-packages/django/conf/__init__.py", line 106, in __init__
django_1    |     mod = importlib.import_module(self.SETTINGS_MODULE)
django_1    |   File "/usr/local/lib/python3.7/importlib/__init__.py", line 127, in import_module
django_1    |     return _bootstrap._gcd_import(name[level:], package, level)
django_1    |   File "/usr/local/src/oar/settings.py", line 186, in <module>
django_1    |     'Invalid GOOGLE_GEOCODING_API_KEY provided, must be set')
django_1    | django.core.exceptions.ImproperlyConfigured: Invalid GOOGLE_GEOCODING_API_KEY provided, must be set
```

Reply:
Yeah, I think that having a geocoding API key is a fair prerequisite. Part of the app would be busted without it.